src/templates/Explore/CommunityPoolsTable/index.tsx

```diff
@@ -23,53 +23,43 @@ import {
   Value as V
 } from '@ui/Modals/ModalViewCoin/styles'
 
-type IPoolsInfosProps = {
-  __typename?: 'Pool' | undefined
+type UnderlyingAssets = {
+  token: {
+    logo?: string | null
+    wraps?: {
+      logo?: string | null
+    } | null
+  }
+}
+
+interface IPoolsInfosProps {
```

Comment:
You changed it to interface for any reason? It's fine for me, just a question

Reply:
This is a standard